x-pack/plugins/infra/server/lib/alerting/log_threshold/log_threshold_executor.test.ts

```diff
@@ -404,127 +404,138 @@ describe('Log threshold executor', () => {
     });
   });
 
-  describe('Results processors', () => {
-    describe('Can process ungrouped results', () => {
-      test('It handles the ALERT state correctly', () => {
-        const alertUpdaterMock = jest.fn();
-        const ruleParams = {
-          ...baseRuleParams,
-          criteria: [positiveCriteria[0]],
-        };
-        const results = {
-          hits: {
-            total: {
-              value: 10,
-            },
-          },
-        } as UngroupedSearchQueryResponse;
-        processUngroupedResults(
-          results,
-          ruleParams,
-          alertsMock.createAlertFactory.create,
-          alertUpdaterMock
-        );
-        // First call, second argument
-        expect(alertUpdaterMock.mock.calls[0][1]).toBe(AlertStates.ALERT);
-        // First call, third argument
-        expect(alertUpdaterMock.mock.calls[0][2]).toEqual([
-          {
-            actionGroup: 'logs.threshold.fired',
-            context: {
-              conditions: 'numericField more than 10',
-              group: null,
-              matchingDocuments: 10,
-              isRatio: false,
-              reason: '10 log entries in the last 5 mins. Alert when > 5.',
-            },
-          },
-        ]);
-      });
-    });
+  // TODO: alertUpdater doesn't exists anymore, but we should keep and update the tests.
```

Comment:
@weltenwort, After the FF I will handle it https://github.com/elastic/kibana/issues/128452

Reply:
Done